### ISSUE TITLE
Incorrect code example in LSP

### ISSUE THREAD

**Issue by Zayon** [jupeter/clean-code-php#55]
See [Liskov substitution principle](https://github.com/jupeter/clean-code-php#liskov-substitution-principle-lsp) **Good** example:

```php
abstract class Shape {
    private $width, $height; // should be protected, not private
    //...
}

class Rectangle extends Shape {
    public function __construct() {
        parent::__construct();
        $this->width = 0; // WRONG can't access $this->width as it's private access
        $this->height = 0; // WRONG can't access $this->height as it's private access
    }
    
    public function setWidth($width) {
        $this->width = $width; // same here
    }
    
    public function setHeight($height) {
        $this->height = $height; // same here
    }
    
    public function getArea() {
        return $this->width * $this->height; // same here
    }
}
```

`$width` and `$height` can't be accessed by `Rectangle` if private in `Shape` abstract class.

**Comment by alexisLefebvre** [comment]
That was fixed in #56.

**Comment by Zayon** [comment]
The PR has been merged
